### ISSUE TITLE
Occasionally, Artist Name is not showing outside of the artist's song list

### ISSUE THREAD

**Issue by konm08** [vfsfitvnm/ViMusic#1179]
### Steps to reproduce the bug

1: Open ViMusic (obviously)
![Screenshot_20230903-090225_ViMusic](https://github.com/vfsfitvnm/ViMusic/assets/62615556/548282e5-2026-4f37-a889-d0a495b0bd39)
2: Search up a song
![Screenshot_20230903-090241_ViMusic](https://github.com/vfsfitvnm/ViMusic/assets/62615556/605dfa2d-f3cb-4bda-bc0e-6aef0199e869)
3: Add said song to your favourites (it may have already showed up glitched at this point)
![Screenshot_20230903-090253_ViMusic](https://github.com/vfsfitvnm/ViMusic/assets/62615556/446b0b42-94c6-4ad5-900c-3e4bc24032c3)

### Expected behavior

Go to your favourites list and see the artist name and track time of the song in their respectable labels.
![Screenshot_20230903-091236_ViMusic](https://github.com/vfsfitvnm/ViMusic/assets/62615556/8005b0b1-d6fb-4eee-b48c-c97f733fc35c)

### Actual behavior

Go to your favourites list, and you may see the artist as "Null" (basically an empty variable) and the track time as the views count.
![Screenshot_20230903-090319_ViMusic](https://github.com/vfsfitvnm/ViMusic/assets/62615556/0d0e3912-a58d-4bb2-96e1-f8e9c85ea8c6)

### ViMusic version

0.5.4

### Android version

Android 13, LineageOS 20-20230613-UNOFFICIAL-nashc

### Additional information

I suspect this might as well be due to the API used, but I might also be wrong.

**Comment by vfsfitvnm** [comment]
Thanks for reporting - I am aware of this bug. It happens because youtube changed the order of the song information, and thus I can't detect the artist anymore, and I wrongly take the view count as the song duration

**Comment by tanmaytez** [comment]
> Thanks for reporting - I am aware of this bug. It happens because youtube changed the order of the song information, and thus I can't detect the artist anymore, and I wrongly take the view count as the song duration

Hey dev, will u be giving this app an update soon?

**Comment by Mirogoal** [comment]
Yeah I'm having this problem too and yall need to fix it and also update the damn app bruh its been 2 years
